<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogRepositionSurface</class>
 <widget class="QDialog" name="DialogRepositionSurface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reposition Surface</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabVolume">
      <attribute name="title">
       <string>Volume Based</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Vertex</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditVertex">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="comboBoxTarget">
           <item>
            <property name="text">
             <string>Intensity</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Coordinate</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Size</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEditSize">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Sigma</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEditSigma">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>2.0</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_6">
           <property name="styleSheet">
            <string notr="true">color:gray;</string>
           </property>
           <property name="text">
            <string>Shift+Left Btn</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="labelHelper">
           <property name="styleSheet">
            <string notr="true">color: gray;</string>
           </property>
           <property name="text">
            <string>Shift+Ctrl+Left Btn</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="lineEditTarget">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditTargetX">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditTargetY">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditTargetZ">
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Gradient direction</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonForceDirectionAuto">
           <property name="text">
            <string>Auto</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string>buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonForceDirectionIn">
           <property name="text">
            <string>In</string>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string>buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonForceDirectionOut">
           <property name="text">
            <string>Out</string>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string>buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxAllowIntersection">
           <property name="text">
            <string>Allow intersection</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>5</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCoordinate">
      <attribute name="title">
       <string>Edit Coordinate</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Vertex</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="lineEditVertex2">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>1</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Coordinate</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="lineEditCoordX">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditCoordY">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditCoordZ">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>1</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QRadioButton" name="radioButtonCoordSurface">
           <property name="text">
            <string>Surface</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonCoordRAS">
           <property name="text">
            <string>RAS</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>5</width>
           <height>5</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSaveAs">
       <property name="text">
        <string>Save As</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonUndo">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Undo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>DialogRepositionSurface</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxTarget</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>DialogRepositionSurface</receiver>
   <slot>OnComboTarget(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonApply</sender>
   <signal>clicked()</signal>
   <receiver>DialogRepositionSurface</receiver>
   <slot>OnApply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSave</sender>
   <signal>clicked()</signal>
   <receiver>DialogRepositionSurface</receiver>
   <slot>OnSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSaveAs</sender>
   <signal>clicked()</signal>
   <receiver>DialogRepositionSurface</receiver>
   <slot>OnSaveAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonUndo</sender>
   <signal>clicked()</signal>
   <receiver>DialogRepositionSurface</receiver>
   <slot>OnUndo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>359</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonCoordRAS</sender>
   <signal>toggled(bool)</signal>
   <receiver>DialogRepositionSurface</receiver>
   <slot>OnCoordinateTypeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonCoordSurface</sender>
   <signal>toggled(bool)</signal>
   <receiver>DialogRepositionSurface</receiver>
   <slot>OnCoordinateTypeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnComboTarget(int)</slot>
  <slot>OnApply()</slot>
  <slot>OnSave()</slot>
  <slot>OnSaveAs()</slot>
  <slot>OnUndo()</slot>
  <slot>OnCoordinateTypeChanged()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
